config/locales/pl.yml

```diff
@@ -49,6 +49,7 @@ pl:
     attend: "Weź udział"
     not_attend: "Anuluj"
     login_before_attend: "Najpierw musisz się zalogować"
+    attend_to_view_attendees_information: "Informacje widoczne tylko dla uczestników"
```

Comment:
@katafrakt is the generated Polish translation ok?

Reply:
It's ok :+1: